src/libgrate/matrix.c

```diff
@@ -143,7 +143,7 @@ void mat4_perspective(struct mat4 *m, float fov, float aspect,
 	m->xx = cotangent / aspect;
 	m->yy = cotangent;
 	m->zz = -(far + near) / depth;
-	m->zw = -1.0f;
-	m->wz = -2.0f * near * far / depth;
+	m->zw = -2.0f * near * far / depth;
+	m->wz = -1.0f;
```

Comment:
My thought was that it has kind of become the canonical OpenGL matrix library. But it being C++ is probably a show-stopper for that, didn't think of that. But I'm a bit confused that this doen't seem to match what GLM does. `Result[3][2]` seems like i would match `.wz`, not `.zw`...